bundles/org.openhab.binding.goecharger/src/main/java/org/openhab/binding/goecharger/internal/handler/GoEChargerV2Handler.java

```diff
@@ -0,0 +1,349 @@
+                key = "psm";
+                if (command instanceof DecimalType) {
+                    var phases = 1;
+                    var help = (DecimalType) command;
+                    if (help.intValue() == 3) {
+                        // set value 2 for 3 phases
+                        phases = 2;
+                    }
+                    value = String.valueOf(phases);
+                }
+                break;
+            case FORCE_STATE:
+                key = "frc";
+                if (command instanceof DecimalType) {
+                    value = String.valueOf(((DecimalType) command).intValue());
+                }
+        }
+
+        if (key != null && value != null) {
+            sendData(key, value);
+        } else {
+            logger.warn("Could not update channel {} with key {} and value {}", channelUID.getId(), key, value);
+        }
+    }
+
+    @Override
+    public void initialize() {
+        // only read needed parameters
+        filter = "?filter=";
+        var declaredFields = GoEStatusResponseV2DTO.class.getDeclaredFields();
+        var declaredFieldsBase = GoEStatusResponseV2DTO.class.getSuperclass().getDeclaredFields();
+
+        for (var field : declaredFields) {
+            filter += field.getAnnotation(SerializedName.class).value() + ",";
+        }
+        for (var field : declaredFieldsBase) {
+            filter += field.getAnnotation(SerializedName.class).value() + ",";
+        }
+        filter = filter.substring(0, filter.length() - 1);
+
+        super.initialize();
+    }
+
+    private String getReadUrl() {
+        return GoEChargerBindingConstants.API_URL_V2.replace("%IP%", StringUtils.trimToEmpty(config.ip)) + filter;
+    }
+
+    private String getWriteUrl(String key, String value) {
+        return GoEChargerBindingConstants.SET_URL_V2.replace("%IP%", StringUtils.trimToEmpty(config.ip))
+                .replace("%KEY%", key).replace("%VALUE%", value);
+    }
+
+    private void sendData(String key, String value) {
+        String urlStr = getWriteUrl(key, value);
+        logger.debug("POST URL = {}", urlStr);
+
+        try {
+            HttpMethod httpMethod = HttpMethod.GET;
+            ContentResponse contentResponse = httpClient.newRequest(urlStr).method(httpMethod)
+                    .timeout(5, TimeUnit.SECONDS).send();
+            String response = contentResponse.getContentAsString();
+
+            logger.debug("{} Response: {}", httpMethod.toString(), response);
+
+            var statusCode = contentResponse.getStatus();
+            if (!(statusCode == 200 || statusCode == 204)) {
+                logger.warn("Could not send data, Response {}, StatusCode: {}", response, statusCode);
+            }
+        } catch (InterruptedException | TimeoutException | ExecutionException | JsonSyntaxException e) {
+            logger.warn("Could not send data: {}, {}", urlStr, e.toString());
+        }
+    }
+
+    /**
+     * Request new data from Go-E charger
```

Comment:
Please use consistent naming of the charger, the readme uses Go-eCharger. Several other pleases too.